content/blogs/2024-04-05-getting-started-with-kestra.md

```diff
@@ -0,0 +1,315 @@
+    type: io.kestra.plugin.notifications.discord.DiscordExecution
+    url: "{{ inputs.discord_webhook_url }}"
+    avatarUrl: "{{ inputs.kestra_logo }}"
+    username: Kestra
+    content: "Total of GitHub Stars: {{ outputs.python_script.vars.output }}"
+```
+
+Let’s execute this and see the outcome!
+
+![full_logs](/blogs/2024-04-05-getting-started-with-kestra/full_logs.png)
+
+Our Python script is executed once the virtual environment is created and the dependencies are installed. The output of this is passed back to Kestra so it can be handed down to our next two tasks. The log outputs our variable correctly and we also see the variable in our Discord channel too with the correct title as well avatar image we defined as an input!
+
+![discord_message](/blogs/2024-04-05-getting-started-with-kestra/discord_message.png)
+
+## Setting up a Trigger
+
+Now that we have everything running, there’s one last step we need to complete this workflow: set up a trigger to execute our flow automatically! As mentioned earlier, you can set flows to execute based on an event, such as a schedule or webhook. For our example, we’re going to use a schedule to run it once every hour. 
+
+To start with, we can use the `triggers` keyword underneath our tasks to specify our schedule. Similar to tasks, each trigger has an `id` and a `type`. With this in mind, we can call our trigger `hour_trigger` and we will want the `Schedule` type. For the `Schedule` type, we will also need to fill in a `cron` property so it knows what schedule to use. 
+
+We can use [crontab.guru](https://crontab.guru) to help us figure out what the correct cron schedule expression would be to run once every hour. This tool is super helpful in visualising what the different expressions mean, as well as with a handy glossary to understand the syntax!
+
+![crontab](/blogs/2024-04-05-getting-started-with-kestra/crontab.png)
+
+This cron schedule expression will execute it at minute 0 of every hour so we can now put that into our cron property in our trigger.
+
+```yaml
+triggers:
+  - id: hour_trigger
+    type: io.kestra.core.models.triggers.types.Schedule
+    cron: 0 * * * *
+```
+
+When we look at our topology view, we can now see our trigger has been correctly recognised. There’s no further actions needed to setup the trigger, it will work as soon as you’ve saved your flow! But it is worth noting that if you want to disable it, you can add a `disabled` property set to true so you don’t have to delete it. Helpfully, you can find all these extra properties through the topology edit view!
+
+![topology](/blogs/2024-04-05-getting-started-with-kestra/topology.png)
+
+With that set up, we now have our fully functioning flow that can make an API request to GitHub through our Python script, output a value from that request to our Kestra logs as well as send it as a Discord notification. And on top of that, it will automatically execute once every hour! To recap, our flow should look like this:
```

Comment:
```suggestion
With that set up, we now have our fully functioning flow that can make an API request to GitHub through our Python script, output a value from that request as well as send it as a Discord notification. And on top of that, it will automatically execute once every hour! To recap, our flow should look like this:
```

nitpick: outputs are stored in the DB  backend + file outputs are stored in internal storage (so not logs)

Reply:
This is referring to the log task in the flow as I left that in to show 3 tasks unless I'm misunderstanding?